requirements.in

```diff
@@ -1,25 +1,24 @@
-pycparser==2.14 # TEMPORARY
+-e git+https://github.com/steve-gregory/billiard.git#egg=billiard # TEMPORARY
 
-Django==1.9.8
+Django==1.10.6
```

Comment:
Note: I think it might be unnecessary to specify exact versions (`==`) in the `.in` files (except in rare circumstances). Something to discuss.